java/test/jmri/jmrix/loconet/LnOpsModeProgrammerTest.java

```diff
@@ -466,6 +466,78 @@ public void testOpsReadLocoNetMode() throws ProgrammerException {
 
      }
 
+     @Test
+     public void testOpsReadLocoNetModeLACKRejected() throws ProgrammerException {
```

Comment:
I am not comfortable with passing an exception up thru the object tree in a test.  

Presumably this is here because some invoked method may throw an exception.  It would be better to place a `try`/`catch` construct around any such method invocation.  If the exception is unexpected in "normal" test behavior, then the `catch` ought to `Assert.fail()` with a message about the unexpected `ProgrammerException`.

Do you have some other reason to pass the `ProgrammerException` to the test infrastructure?

Reply:
@devel-bobm I simply copied the existing test case and went from there.  Almost all the test cases in this file throw that exception.

Reply:
Ok - leave it as-is, and we can worry about it at another time.  

Would need to determine which method can throw the exception, then put the `try`/`catch` structure around that method invocation, and add appropriate code to give reasonable message upon the unexpected exception.  That's more effort than I can spend at this time...